client/src/actions/holidays.js

```diff
@@ -0,0 +1,90 @@
+import axios                from 'axios';
+import cookie               from 'react-cookie';
+
+const API_URL = `${window.location.origin}/api/v1/holidays`;
+
+const headers = {};
+headers['Authorization'] = `Bearer ${cookie.load('token')}`;
+if (cookie.load('workspace')) {
```

Comment:
holidays are not workspace related, do we really need it here?

Reply:
problem solved